ksql-engine/src/test/java/io/confluent/ksql/integration/JoinIntTest.java

```diff
@@ -107,10 +123,62 @@ public void shouldLeftJoinOrderAndItems() throws Exception {
     }, 60000, "failed to complete join correctly");
   }
 
+
+  @Test
+  public void shouldLeftJoinOrderAndItemsJsonAvro() throws Exception {
```

Comment:
`JsonAvro`? i only see avro names in here

Reply:
Refactored this test for both avro and json.